### PR TITLE
lookup since 24 hours, not days [disable e2e tests]

### PR DESCRIPTION
**Pull request by ckiosidis**
# Hey, I just made a Pull Request!

- improve the query performance of list executions by limiting the start date
- disable flyte execution E2E tests for now (will revisit in another PR)

## Checklist for PR author(s)

- [ ] Changes are covered by unit test
- [ ] All tests pass
- [ ] Code coverage check passes
- [ ] Error handling is tested
- [ ] Errors are handled at the appropriate layer
- [ ] Errors that cannot be handled where they occur are propagated
- [ ] (optional) Changes are covered by system test
- [ ] Relevant documentation updated
- [ ] This PR has NO breaking change to public API
- [ ] This PR has breaking change to public API and it is documented

## Checklist for PR reviewer(s)

- [ ] This PR has been incorporated in release note for the coming version
- [ ] Risky changes introduced by this PR have been all considered

### CODE REVIEW

File: styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java [hunk 1]
```diff
@@ -84,7 +84,7 @@ public class FlyteAdminClientRunner implements FlyteRunner {
   @VisibleForTesting static final String STYX_WORKFLOW_INSTANCE_ANNOTATION = "styx-workflow-instance";
   @VisibleForTesting static final String STYX_EXECUTION_ID_ANNOTATION = "styx-execution-id";
   @VisibleForTesting static final Duration TERMINATION_GRACE_PERIOD = Duration.ofMinutes(10);
-  private static final Duration TERMINATION_LOOKUP_SINCE = Duration.ofDays(24);
```
Comment (honnix) [inline styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java:87]: 😮 Are are sure this was not a bug? `24 days` looks too much to be true, and the number `24` feels like hours anyway.

**Review by honnix — APPROVED** [review]

**Comment by honnix** [comment]
Please write a test case for it.

File: styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java (hunk 1 above)
Comment (ckiosidis) [reply styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java:87]: yeah i think it was unintentional

**Comment by ckiosidis** [comment]
ok

**Comment by honnix** [comment]
> Please write a test case for it.

We can do it in a separated PR if we want to ship this asap.

File: styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/EndToEndTestBase.java [hunk 2]
```diff
@@ -110,7 +110,7 @@ public class EndToEndTestBase {
 
   private static final String FLYTE_DOMAIN = "development";
   private static final String FLYTE_LAUNCH_PLAN_NAME = "morning_greeting";
-  private static final String FLYTE_LAUNCH_PLAN_VERSION = "04e346ca5f43fc2778259d77a2d9f64ce42b2a27";
+  private static final String FLYTE_LAUNCH_PLAN_VERSION = "v0";
```
Comment (honnix) [inline styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/EndToEndTestBase.java:113]: Let's restore this change?

**Review by honnix — APPROVED** [review]

File: styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/EndToEndTestBase.java (hunk 2 above)
Comment (ckiosidis) [reply styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/EndToEndTestBase.java:113]: sure, but these tasks do not exist right now

**Pull request merged**